README.md

```diff
@@ -31,6 +31,7 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Auth0 | All | Full remote okay | [Link](https://auth0.com/careers) | ? |
 | bill.com |  | Some areas | [Link](https://www.bill.com/about-us/jobs) | ? |
 | Bishop Fox |  | Some areas | [Link](https://www.bishopfox.com/jobs/) | ? |
+| Bitdefender | Some | Full remote US | [Link](https://www.bitdefender.com/company/job-opportunities/) | ? |
```

Comment:
Example listing: https://bitdefender.csod.com/ux/ats/careersite/1/home/requisition/401?c=bitdefender

Reply:
I can't spot any sign of remote working ..